### PR TITLE
[TEST] use EXPECT_RANGE_EQ in get_test...

### PR DESCRIPTION
**Pull request by marehr**
Part of seqan/product_backlog#83.

### CODE REVIEW

File: test/unit/range/views/get_test.cpp [hunk 1]
```diff
@@ -74,80 +59,53 @@ TEST(view_get, advanced)
                                                    {'C'_dna4, seqan3::mask::UNMASKED},
                                                    {'G'_dna4, seqan3::mask::MASKED},
                                                    {'T'_dna4, seqan3::mask::UNMASKED}};
-    std::vector<seqan3::masked<seqan3::dna4>> functor0 = seqan3::views::get<0>(t) | seqan3::views::to<std::vector>;
-    EXPECT_EQ(cmp0, functor0);
 
-    std::vector<seqan3::phred42> cmp1{seqan3::phred42{0}, seqan3::phred42{1}, seqan3::phred42{2}, seqan3::phred42{3}};
-    std::vector<seqan3::phred42> functor1 = seqan3::views::get<1>(t) | seqan3::views::to<std::vector>;
-    EXPECT_EQ(cmp1, functor1);
+    EXPECT_RANGE_EQ(cmp0, seqan3::views::get<0>(t));
```
Comment (Irallia) [inline test/unit/range/views/get_test.cpp:63]: As there is no `cmp00` anymore you could rename `cmp0` to `cmp` or maybe maybe use a more explanatory name as `functor_compared`.

File: test/unit/range/views/istreambuf_test.cpp [hunk 2]
```diff
@@ -46,39 +45,32 @@ INSTANTIATE_TYPED_TEST_SUITE_P(iterator_fixture, iterator_fixture, test_type, );
 
 TEST(view_istreambuf, basic)
 {
+    using namespace std::literals;
```
Comment (Irallia) [inline test/unit/range/views/istreambuf_test.cpp:48]: Is this for using this `operator""sv` thing?

**Review by Irallia — APPROVED** [review]
Looks very good!
However, I noticed that the tests use a lot of meaningless variables like `cmp0`, `t` and `is`. Maybe you want to change something about that?
And I have a question about the using literal stuff. Maybe you want to add a short documentation notice?

I approve either way, you can decide if you want to rename the variables or if that is too much effort.

File: test/unit/range/views/istreambuf_test.cpp (hunk 2 above)
Comment (eseiler) [reply test/unit/range/views/istreambuf_test.cpp:48]: Yes, we also want to do `seqan3::literals`. See https://github.com/seqan/product_backlog/issues/287#issuecomment-775223706 for some reasoning

**Review by eseiler — CHANGES_REQUESTED** [review]
Open point about renaming, otherwise LGTM

File: test/unit/range/views/istreambuf_test.cpp (hunk 2 above)
Comment (Irallia) [reply test/unit/range/views/istreambuf_test.cpp:48]: Ah so we can use it for all operator""... stuff. Nice!

**Comment by marehr** [comment]
> Open point about renaming, otherwise LGTM

😭 I don't want to rename everything.

File: test/unit/range/views/get_test.cpp (hunk 1 above)
Comment (marehr) [reply test/unit/range/views/get_test.cpp]: I renamed the stuff in the get test.

**Review by eseiler — APPROVED** [review]

**Pull request merged**